src/nethsec/firewall/__init__.py

```diff
@@ -1200,10 +1216,13 @@ def list_rules(uci, rule_type = None) -> list:
             else:
                 if rule_type =='forward' and is_forward_rule(rule):
                    rules.append(enrich_rule(uci, rule))
+                   rule['active_zone'] = is_zone(uci, rule['src'])
```

Comment:
In forward zones we must check both destination and source zone.
```suggestion
                   rule['active_zone'] = is_zone(uci, rule['src']) and is_zone(uci, rule['dst']) 
```